modules/ocm-networking-tab-concept.adoc

```diff
@@ -5,18 +5,24 @@
 [id="ocm-networking-tab_{context}"]
 = Networking tab
 
-The **Networking** tab provides a control plane API endpoint as well as the default application router. Both the control plane API endpoint and the default application router can be made private by selecting the respective box below each of them.
+The **Networking** tab provides a control plane API endpoint as well as the default application router. Both the control plane API endpoint and the default application router can be made private by selecting the respective box below each of them. If applicable, you can also find your virtual private cloud (VPC) details on this tab.
```

Comment:
```suggestion
The **Networking** tab provides a control plane API endpoint as well as the default application router. Both the control plane API endpoint and the default application router can be made private by selecting the respective box below each title. If applicable, you can also find your virtual private cloud (VPC) details on this tab.
```
Not sure if "title" or "label" or something else entirely would be the best word to substitute here. I just think "each of them" is a little vague.